aiida/calculations/arithmetic/add.py

```diff
@@ -7,26 +7,36 @@
 # For further information on the license, see the LICENSE.txt file        #
 # For further information please visit http://www.aiida.net               #
 ###########################################################################
-"""`CalcJob` implementation to add two numbers using bash for testing and demonstration purposes."""
+# yapf: disable
+"""`CalcJob` implementation to add two numbers using bash for testing and demonstration purposes.
+
+.. warning::
+
+    This implementation is used directly in the documentation as a literal-include, which means that if any part of this
+    code is changed, the snippets in the file `docs/source/howto/codes.rst` have to be checked for consistency.
```

Comment:
I don't think this should be part of the doc-string, but instead an Python comment. Otherwise it shows up [here](https://258-77234579-gh.circle-artifacts.com/0/html/reference/apidoc/aiida.calculations.arithmetic.html#module-aiida.calculations.arithmetic.add) as part of the reference documentation, which I don't think is intended.